doc/howto/report-bugs.rst

```diff
@@ -0,0 +1,58 @@
+.. _report-bugs:
+
+How to report a problem
+***********************
+
+We always hope, of course, that every install with the server installer
+succeeds. But reality doesn't always work that way and there will sometimes be
+failures of various kinds. This section explains the most useful way to report
+any failures so that we can fix the bugs causing them, and we'll keep the topic
+up to date as the installer changes.
+
+Update Subiquity
+================
+
+The first thing to do is to update your Subiquity snap. Not only because we
+fix issues that cause failures over time but also because we've been working on
+features to make failure reporting easier.
+
+<command here? What if the snap isn't installed?>
```

Comment:
Can we include the relevant commands here?

Reply:
My sense for the docs that the only case we should talk about updating snaps is if there is a new stable version released, which means the snap will be offered anyhow.

The only cases why a stable snap would not be offered are:
1) offline installs or network issues
2) old install ISOs that no longer receive newer subiquity snaps being offered (bionic)

My sense is that we can remind people here of the purpose of the refresh page, and if they are offline to try again online and see if a newer snap is offered.